netrd/reconstruction/template.py

```diff
@@ -0,0 +1,72 @@
+"""
+<file_name>.py
```

Comment:
Please add a line of ----------------- under the name. This is important if we ever want to automatically generate documentation for the package, as it will be interpreted as Markdown.